Whisky/Utils/Logger.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Logger.swift
+//  Whisky
+//
+//  Created by Isaac Marovitz on 08/06/2023.
+//
+
+import Foundation
+import OSLog
+
+class Log {
+    static let logsFolder = FileManager.default.urls(for: .libraryDirectory,
+                                                    in: .userDomainMask)[0]
+        .appendingPathComponent("Logs")
+        .appendingPathComponent("Whisky")
+
+    let fileHandle: FileHandle
+    let logger: Logger
+
+    init?() {
```

Comment:
Instead of `init?`, you're calling this from a throwing function.
Why not make this a throwing initialiser too?

```suggestion
    init() throws {
```

Reply:
It seemed a nice API to not have to handle errors whenever we need to use a Logger